### DISCUSSION TITLE
How to make the absolute value of MOMENT_Z an Objective?

### DISCUSSION THREAD

**Discussion by 1Northward1** [su2code/SU2#1770] [Q&A]
Hi,

I want to reduce the absolute value of MOMENT_Z of an airfoil, cause the original airfoil has a high pitching-down moment.
I have tried to add fabs(MOMENT_Z) to the .cfg file: OPT_OBJECTIVE= fabs(MOMENT_Z), but it failed.
How to make the absolute value of MOMENT_Z  an objective?

I would appreciate it if you help me.

Best regards,
Mars

**Comment by pcarruscag** [comment]
Hi,
You need to use the custom objective function, see https://github.com/su2code/SU2/blob/master/TestCases/user_defined_functions/lam_flatplate.cfg

**Reply by pcarruscag** [reply]
I don't know if this will work with the shape_optimization.py script

**Reply by 1Northward1** [reply]
Thank you very much! I'll try it.
